### PR TITLE
[10.0][ADD] sale_product_country_restriction

### PR DESCRIPTION
**Pull request by rousseldenis**
Manages product embargoes in sale workflows.

Depends on :
* [x] product_embargo https://github.com/OCA/product-attribute/pull/412

Supersedes #672

### CODE REVIEW

**Comment by rousseldenis** [comment]
Needs review

**Comment by rousseldenis** [comment]
@Cedric-Pigeon

File: sale_product_embargo/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Product Embargo',
+    'summary': """
+        Implements product embargoes on sale workflow""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu',
```
Comment (nikul-serpentcs) [inline sale_product_embargo/__manifest__.py]: @rousseldenis Change URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

**Review by nikul-serpentcs — COMMENTED** [review]
Minor change

**Review by Cedric-Pigeon — APPROVED** [review]

**Comment by rafaelbn** [comment]
Hi @rousseldenis please rebase thanks

**Comment by rafaelbn** [comment]
Ready to rebase @rousseldenis

**Comment by rafaelbn** [comment]
Hi @rousseldenis if you rebase I cant test it

**Pull request closed without merging**